analyze-documents-ai-docintelligence/manifest.json

```
@@ -1,5 +1,5 @@
 {
-    "title": "Document Intelligence: Interact and Analyze complex documents using Document Intelligence and Azure OpenAI",
+    "title": "Document Intelligence: Analyze complex documents using Document Intelligence and Azure OpenAI",
     "description": "This workflow uses Forms Recognizer connector to covert complex documents into markdown and Azure Open AI connector to analyze the image content, and provides response to specific questions, based on that image.",
     "prerequisites": "To use this template, you need to configure an Azure Open AI instance and an embedding model. Learn more [here](https://github.com/Azure/logicapps/tree/master/LogicApps-AI-RAG-Demo).You also need to [create an Azure AI service or Document Intelligent resource](https://learn.microsoft.com/azure/ai-services/document-intelligence/quickstarts/get-started-sdks-rest-api?view=doc-intel-4.0.0&pivots=programming-language-csharp), and use the keys and endpoint to create connection. ",
```

- No hardcoded doc links. Use FW links (go.microsoft.com links, not aka.ms)
- Use descriptive link text, not "here".

```suggestion
    "prerequisites": "To use this template, you need to configure an Azure Open AI instance and an embedding model. For more information, see [Create a chat with your data logic app project](https://github.com/Azure/logicapps/tree/master/LogicApps-AI-RAG-Demo).You also need to [create an Azure AI service or Document Intelligent resource](https://go.microsoft.com/fwlink/?linkid=2302318), and use the keys and endpoint to create a connection.",
```